Source/Events.Processing/Projections/IValidateOccurredFormat.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Runtime.Projections.Store.Definition;
+
+namespace Dolittle.Runtime.Events.Processing.Projections;
+
+/// <summary>
+/// Defines a validator for <see cref="OccurredFormat"/>.
+/// </summary>
+public interface IValidateOccurredFormat
+{
+    /// <summary>
+    /// Checks whether the given <see cref="OccurredFormat"/> is valid.
```

Comment:
```suggestion
    /// Checks whether the given <see cref="OccurredFormat"/> is valid, meaning it can be used to format <see cref="DateTimeOffset"/> to a string.
```